tests/conftest.py

```diff
@@ -375,13 +376,12 @@ def testsuite_properties(record_testsuite_property, pytestconfig):
     Configures custom testsuite properties for junit xml
     """
     noobaa_sa_rpm_name = get_noobaa_sa_rpm_name()
-    time_now = datetime.now()
-    time_nums = time_now.strftime("%Y%m%d_%H%M%S")
-    job_name = f"noobaa-sa-{noobaa_sa_rpm_name}-{time_nums}"
-    record_testsuite_property("rp_noobaa_sa_rpm_name", noobaa_sa_rpm_name)
+    noobaa_sa_version_string = get_noobaa_sa_version_string(noobaa_sa_rpm_name)
+    job_name = f"noobaa-sa-{noobaa_sa_version_string}"
```

Comment:
I would have it here rather as rpm_name variable,
and again in lauch_name - we want only x.y without z 

And whole version - also with patch like this: noobaa-core-5.17.0-20241026.el9.x86_64

Should be added as new attribute called rp_rpm_name and we can have also rp_rpm_version which will have e.g. 5.17.0 which will allow us to filter execution for this specific x.y.z which will be useful I think.